src/components/Match/MatchGraph.jsx

```diff
@@ -49,9 +49,12 @@ const drawGraphs = (props, id) => {
       },
       tooltip: {
         contents(d, defaultTitleFormat, defaultValueFormat, color) {
```

Comment:
It provides the level of the experience via Line24 on this file. If we removed it we would lose that functionality.